### PR TITLE
Vcd speaker

### PR DESCRIPTION
**Pull request by victordibia**
- Update audio play - use sound-player library that allows users specify speaker `deviceid`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "object.pick": "^1.2.0",
     "semaphore": "^1.0.5",
     "sleep": "^5.0.0",
+    "sound-player": "^1.0.4",
```
Comment (jweisz) [inline package.json]: we also need to remove `aplay` from this list, right?

File: lib/tjbot.js [hunk 2]
```diff
@@ -1206,13 +1207,22 @@ TJBot.prototype.play = function(soundFile) {
...
+        if (self._soundplayer == undefined) {
             reject(new Error("unable to play audio, TJBot hardware doesn't include a \"speaker\""));
             return;
         }
 
-        // play the audio
-        var player = new self._aplay();
+        // initialize soundplayer lib
+        var speakerOptions = {
+            "filename": soundFile,
```
Comment (jweisz) [inline lib/tjbot.js]: style question -- does `"filename"` need to be in quotes?

File: package.json (hunk 1 above)
Comment (victordibia) [inline package.json]: Yes, its no longer needed.

File: lib/tjbot.js (hunk 2 above)
Comment (victordibia) [inline lib/tjbot.js]: Yeah. Doesnt. Updated.

File: package.json (hunk 1 above)
Comment (victordibia) [inline package.json]: removed.

**Pull request merged**